src/MainPage/MainPage.js

```diff
@@ -57,10 +50,27 @@ function MainPage() {
         console.error('Fetch operation failed:', error);
       });
   }
+  
+  const updateEnergy = () => {
```

Comment:
might consider renaming to better reflect feeding action, but it works which is really the goal. I like the how we dynamically call the pokemon's id in the fetch